sdk/communication/azure-communication-chat/azure/communication/chat/_models.py

```diff
@@ -284,3 +288,63 @@ def __init__(
         # type: (...) -> None
         self.chat_thread = kwargs['chat_thread']
         self.errors = kwargs.get('errors', None)
+
+class ChatRetentionPolicy(object):
+    """Data retention policy for auto deletion. It's not updatable after creation.
+
+    You probably want to use the sub-classes and not this class directly. Known sub-classes are:
+    ThreadCreationDateRetentionPolicy
+
+    All required parameters must be populated in order to send to Azure.
+
+    :ivar kind: Retention Policy Type. Required. "threadCreationDate"
+    :vartype kind: str or ~azure.communication.chat.models.RetentionPolicyKind
+    """
+
+    _validation = {
+        "kind": {"required": True},
+    }
+
+    _attribute_map = {
+        "kind": {"key": "kind", "type": "str"},
+    }
+
+    _subtype_map = {"kind": {"threadCreationDate": "ThreadCreationDateRetentionPolicy"}}
```

Comment:
We should be able to remove these properties (`_validatation`, `_attribute_map`, _subtype_map`)